### PR TITLE
FH-3649 - Change the cmd $fhc version to follow the V3 standard

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3649

**Changes**
- Move to V3 standard
- Move methods used just locally for local functions
- Fix the command to return the value into json format
- Refectory the code

**Steps to Verify**
- fhc version
- fhc version --json
 
**Local Tests**
```
Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js version --json
{
  "plataformVersion": "3.18.2",
  "latestNpmVersion": "2.18.0-896",
  "localFHCVersion": "3.0.4-BUILD-NUMBER",
  "isUpToDate": true,
  "isLatest": false
}
Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js version 
FHC Version: 3.0.4-BUILD-NUMBER (Up to date)
RHMAP Version: 3.18.2
Camilas-MacBook-Pro:fh-fhc cmacedo$ 
```

### CODE REVIEW

File: lib/cmd/fh3/version.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    });
+  },
+  'postCmd': function(params, response, cb) {
+    if (!params.json) {
+      return cb(null, createTxtMessage(response, ini.get('fhclatest')));
+    }
+    return cb(null, createJsonMessage(response));
+  },
+  checkFHCUpToDate: function(cb) {
+    var cachedLatest = ini.get('fhclatest');
```
Comment (wtrocki) [inline lib/cmd/fh3/version.js]: In my personal opinion caching this doesn't make sense. Original intention of the author was to enable this command while being offline, but I think that we can just skip that information if request to npm failed.

File: lib/cmd/fh3/version.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      isUpToDate = semver.gte(cleanupVersion(localV), cleanupVersion(latestV));
+
+    self.latest = {
+      version: latestV,
+      is: isUpToDate,
+      ts: new Date().getTime(),
+      current: fhc._version
+    };
+
+    // Cache the result
```
Comment (wtrocki) [inline lib/cmd/fh3/version.js]: Makes no sense to update that as usability of this cache is almost 0.

**Comment by wtrocki** [comment]
Let's be real here - how many times a day you will check version?
Doing network request to npm is relatively low and you always want to know the real value not cache. Any `npm install` command doing 💯  of them to get each package.

**Comment by camilamacedo86** [comment]
@wtrocki fair enough good point !! Done. I just remove it and keep the original code for get the value from the server. 👍

**Comment by camilamacedo86** [comment]
@wtrocki Please, could you check now? It is really better !!!

**Comment by wtrocki** [comment]
Verified: 
```
 fhc version                                                                                                            
FHC Version: 3.0.4-BUILD-NUMBER (Up to date)
RHMAP Version: development
```

**Comment by camilamacedo86** [comment]
@wtrocki really Tks !!! Your review here was a great work and helped to much!!!

**Pull request merged**